relay-server/src/actors/project_cache.rs

```diff
@@ -623,10 +797,33 @@ impl Handler<FlushBuckets> for ProjectCache {
             return;
         }
 
-        EnvelopeManager::from_registry().send(SendMetrics {
-            buckets,
-            scoping,
-            partition_key,
-        });
+        // Check rate limits:
+        let quotas = project_state
+            .config
+            .quotas
+            .iter()
+            .filter(|q| q.categories.contains(&DataCategory::TransactionProcessed))
+            .map(Quota::clone)
+            .collect::<Vec<_>>();
+        let processing_enabled = config.processing_enabled();
+        let rate_limits = project.rate_limits().clone();
+        use futures::{FutureExt, TryFutureExt};
```

Comment:
Do we actually want to remove the import here and move it to the top of the file?